.github/workflows/copyright-header-check.yaml

```diff
@@ -10,4 +10,4 @@ jobs:
       - name: Checkout repo
         uses: actions/checkout@v1
       - name: Run test_copyright.py
-        run: python3 examples_utils/testing/test_copyright.py
+        run: python3 graphcore_cloud_tools/testing/test_copyright.py
```

Comment:
these can stay on examples-utils as we're going to strip it out I would think

Reply:
True, I did global search/replace but forgot about these